src/user/MOM_wave_interface.F90

```diff
@@ -353,40 +360,40 @@ subroutine MOM_wave_interface_init(time, G, GV, US, param_file, CS, diag )
       call MOM_error(FATAL,'Check WAVE_METHOD.')
     end select
 
-  case (DHH85_STRING)!Donelan et al., 1985 spectrum
-    WaveMethod = DHH85
+  case (DHH85_STRING) !Donelan et al., 1985 spectrum
+    CS%WaveMethod = DHH85
     call MOM_error(WARNING,"DHH85 only ever set-up for uniform cases w/"//&
                            " Stokes drift in x-direction.")
-    call get_param(param_file,mdl,"DHH85_AGE_FP",WaveAgePeakFreq,   &
+    call get_param(param_file, mdl, "DHH85_AGE_FP", CS%WaveAgePeakFreq, &
          "Choose true to use waveage in peak frequency.", &
          units='', default=.false.)
-    call get_param(param_file,mdl,"DHH85_AGE",WaveAge,   &
+    call get_param(param_file, mdl, "DHH85_AGE", CS%WaveAge, &
          "Wave Age for DHH85 spectrum.", &
          units='', default=1.2)
-    call get_param(param_file,mdl,"DHH85_WIND", WaveWind, &
+    call get_param(param_file,mdl,"DHH85_WIND", CS%WaveWind, &
          "Wind speed for DHH85 spectrum.", &
          units='m s-1', default=10.0, scale=US%m_s_to_L_T)
-    call get_param(param_file,mdl,"STATIC_DHH85",StaticWaves,   &
+    call get_param(param_file,mdl,"STATIC_DHH85", CS%StaticWaves, &
          "Flag to disable updating DHH85 Stokes drift.", &
           default=.false.)
   case (LF17_STRING)!Li and Fox-Kemper 17 wind-sea Langmuir number
-    WaveMethod = LF17
+    CS%WaveMethod = LF17
   case default
     call MOM_error(FATAL,'Check WAVE_METHOD.')
   end select
 
   ! Langmuir number Options
-  call get_param(param_file, mdl, "LA_DEPTH_RATIO", LA_FracHBL,              &
-         "The depth (normalized by BLD) to average Stokes drift over in "//&
-         "Langmuir number calculation, where La = sqrt(ust/Stokes).",       &
-         units="nondim",default=0.04)
-  call get_param(param_file, mdl, "LA_MISALIGNMENT", LA_Misalignment,    &
+!  call get_param(param_file, mdl, "LA_DEPTH_RATIO", CS%LA_FracHBL,              &
+!         "The depth (normalized by BLD) to average Stokes drift over in "//&
+!         "Langmuir number calculation, where La = sqrt(ust/Stokes).",       &
+!         units="nondim",default=0.04)
```

Comment:
So far as I can tell you moved this call up so the parameter gets set even in statistical mode.  In that case do we need to keep this around here as commented code?

Reply:
Thanks for pointing this out.  This duplicated code has now been removed.